railties/lib/rails/generators/rails/app/templates/bin/brakeman.tt

```diff
@@ -1,4 +1,6 @@
 require "rubygems"
 require "bundler/setup"
 
+ARGV.unshift("--ensure-latest")
```

Comment:
This file is just the generated binstub by bundler, we should not change its content. If this is a better default, we should be wither changing Breakman to always use it.

If we can't, let's pass it in the ci.yml file.

Reply:
@rafaelfranca I agree with your concerns, however default Rails binstubs are not equal to those which are generated by `bundler`. Please take a look at file that is generated by `bundle binstubs rubocop`.

```
ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)

bundle_binstub = File.expand_path("bundle", __dir__)

if File.file?(bundle_binstub)
  if File.read(bundle_binstub, 300).include?("This file was generated by Bundler")
    load(bundle_binstub)
  else
    abort("Your `bin/bundle` was not generated by Bundler, so this binstub cannot run.
Replace `bin/bundle` by running `bundle binstubs bundler --force`, then run this command again.")
  end
end

require "rubygems"
require "bundler/setup"

load Gem.bin_path("rubocop", "rubocop")
```

At the same time [default Rails](https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/bin/rubocop.tt) `bin/rubocop` has the default param which gave me the idea. 

```
require "rubygems"
require "bundler/setup"

# explicit rubocop config increases performance slightly while avoiding config confusion.
ARGV.unshift("--config", File.expand_path("../.rubocop.yml", __dir__))

load Gem.bin_path("rubocop", "rubocop")
```